WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanel.m

```diff
@@ -27,6 +29,35 @@ - (void)beginSession
 {
     [Mixpanel sharedInstanceWithToken:[WordPressComApiCredentials mixpanelAPIToken]];
     [self refreshMetadata];
+    [self flagIfUserHasSeenLegacyEditor];
+}
+
+- (void)flagIfUserHasSeenLegacyEditor
+{
+    NSUserDefaults *standardDefaults = [NSUserDefaults standardUserDefaults];
+    if ([standardDefaults boolForKey:CheckedIfUserHasSeenLegacyEditor]) {
+        return;
+    }
+    
+    NSInteger sessionCount = [self sessionCount];
+    if ([self didUserCreateAccountOnMobile]) {
+        // We want to differentiate between users who created pre 4.6 and those who created after and the way we do this
+        // is by checking if the editor is enabled. The editor would only be enabled for users who created an account after 4.6.
+        [self setSuperProperty:@"seen_legacy_editor" toValue:@(![WPPostViewController isNewEditorEnabled])];
```

Comment:
Just to avoid typos causing issue in tracking, I'd recommend defining a constant for the value @"seen_legacy_editor".